### PR TITLE
Some packaging suggestions

### PR DESCRIPTION
**Pull request by fgregg**
See inline comments

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,11 +4,12 @@
 
 Run tests in transactions using pytest, Flask, and SQLAlchemy.
 '''
-import os
 from setuptools import setup
 
-with open(os.path.join('requirements', 'main.txt')) as reqfile:
-    requirements = reqfile.read().splitlines()
```
Comment (fgregg) [inline setup.py]: requirements.txt are great for fixing dependencies that you know you will work for an app, but are not the best choice for a library for two reasons

1. you actually want to support the widest range of dependencies possible (counter to what you want to do with an app)
2. keeping your requirements in setup.py reduces some complexity. You could keep this approach but you would need to add 'main.txt' to the MANIFEST.in file.

File: setup.py [hunk 2]
```diff
@@ -4,11 +4,12 @@
...
+
+def readme():
+    with open('README.md') as f:
+        return f.read()
```
Comment (fgregg) [inline setup.py]: You got a great README. if you do this, it will be rendered nicely on pypi.org when you publish there.

File: setup.py [hunk 3]
```diff
@@ -18,12 +19,15 @@
...
+    long_description=readme(),
     license='MIT',
 
     packages=['transactions'],
-    install_requires=requirements,
-
+    install_requires=[pytest>=3.2.1,
+                      pytest-mock>=1.6.2,
+                      SQLAlchemy>=1.2.2
+                      Flask-SQLAlchemy>=2.3],
```
Comment (fgregg) [inline setup.py]: Are these the all the minimum versions that will work with your library?

File: setup.py [hunk 4]
```diff
@@ -18,12 +19,15 @@
...
+    long_description=readme(),
     license='MIT',
 
     packages=['transactions'],
-    install_requires=requirements,
-
+    install_requires=['pytest>=3.2.1',
+                      'pytest-mock>=1.6.2',
+                      'SQLAlchemy>=1.2.2',
+                      'Flask-SQLAlchemy>=2.3'],
```
Comment (fgregg) [inline setup.py]: Are these the minimum versions of the dependencies this library requires?

Comment (jeancochrane) [reply setup.py]: That's a great question -- I'm not entirely sure. Is there a principled approach to testing this, or should I do some trial and error?

File: setup.py (hunk 2 above)
Comment (jeancochrane) [reply setup.py]: Great point!

File: setup.py (hunk 4 above)
Comment (fgregg) [reply setup.py]: I don't know of a principled approach.

**Pull request merged**